dashboard/app/views/parent_mailer/parent_permission_confirmation.html.haml

```diff
@@ -5,7 +5,7 @@
   Thank you for approving your child's use of a Code.org® account (or adding a personal login to an existing Code.org account used in school).  By approving your child's account, you agree to the Code.org
   %a{href:"https://code.org/tos"}Terms of Service
   and
-  %a{href:"https://code.org/privacy"}Privacy Policy.
+  %a{href:"https://code.org/privacy-nov2021#protecting-childrens-privacy"}Privacy Policy.
```

Comment:
Wouldn't it be nice to move the privacy URL to `config.yml.erb` to make it easier to configure in the future?

Reply:
Agreed, it'd be nice to have some repository for these sorts of reused strings. Since we don't seem to have that pattern yet though, I think I'll go ahead with this PR so we can knock this item off the list of remaining CPA tickets.